Ndev 2478 add send erc20 scenario

romanova-natasha opened the pull request:
fix contract module to work with contract instance in load test: remove hardcoded logic, add custom signer to contract obj, minor refactoring

romanova-natasha committed:
add signer to contract instance

add signer to contract instance

romanova-natasha committed:
get gas price and gas limit in txn method before do txn

get gas price and gas limit in txn method before do txn

romanova-natasha committed:
add method: fill tx data for a specific contract method

add method: fill tx data for a specific contract method

kristinaNikolaevaa commented on contract.go:
```diff
@@ -34,18 +36,46 @@ func (c *Contract) Txn(method string, opts TxnOpts, args ...interface{}) (string
+	return c.SignerAddress
+}
+
+func (c *Contract) FillInput(abiString string, method string) []byte {
```
this function should contain function parameters as I understand

romanova-natasha replied:
yes, it was not full
I completed it with full input calculation for given method and args values

romanova-natasha committed:
fix fill tx input method, add max attempts param to receipt waiter function, move gas, gasPrice, nonce calculation to sendRawTransaction function

fix fill tx input method, add max attempts param to receipt waiter function, move gas, gasPrice, nonce calculation to sendRawTransaction function
